docs/development.rst

```diff
@@ -163,6 +163,32 @@ virtual env and run::
 
   pip install -r requirements.d/development.txt
 
+Building on Windows
++++++++++++++++++++
+
+Install scoop: https://scoop.sh/
+
+install MSYS2: ``scoop install msys2``
+
+Use `Mingw64-w64 64bit Shell`.
+In Windows shell activate mingw in the repo root folder: ``mingw``
+
+Then install dependencies
+
+  pacman -S mingw-w64-x86_64-python3 git mingw-w64-x86_64-lz4 mingw-w64-x86_64-python3-pip \
+  mingw-w64-x86_64-cython mingw-w64-x86_64-gcc mingw-w64-x86_64-ntldd-git \
+  mingw-w64-x86_64-pkg-config mingw-w64-x86_64-openssl openssl-devel
```

Comment:
can you sort that into one group per line:
- python stuff
- compression and crypto stuff
- tools

Reply:
how about:
- https://packages.msys2.org/base/mingw-w64-zstd
- https://packages.msys2.org/base/mingw-w64-xxhash

we have that stuff bundled (we also have lz4 bundled), but maybe it gets updated more frequently like that?

Reply:
I'll correct this.